src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowRuleEntity.groovy

```diff
@@ -0,0 +1,159 @@
+package org.openkilda.functionaltests.helpers.model
+
+import org.openkilda.messaging.info.rule.FlowApplyActions
+import org.openkilda.messaging.info.rule.FlowCopyFieldAction
+import org.openkilda.messaging.info.rule.FlowEntry
+import org.openkilda.messaging.info.rule.FlowInstructions
+import org.openkilda.messaging.info.rule.FlowMatchField
+import org.openkilda.messaging.info.rule.FlowSetFieldAction
+import org.openkilda.messaging.info.rule.FlowSwapFieldAction
+
+import groovy.transform.Canonical
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+
+@ToString(includeNames = true)
+@Canonical
+@EqualsAndHashCode(excludes = 'durationSeconds, durationNanoSeconds, packetCount, byteCount')
+class FlowRuleEntity {
```

Comment:
Added a new testing entity FlowRuleEntity that allows us to compare rules on the switch as usage of the FlowEntry excludes some required fields from verification("idleTimeout", "hardTimeout", "version"). 
```
@EqualsAndHashCode(exclude = {"durationSeconds", "durationNanoSeconds", "packetCount", "byteCount",
        "idleTimeout", "hardTimeout", "version"})
FlowEntry
```